pom.xml

```diff
@@ -52,6 +52,17 @@
 		<url>https://github.com/jenkinsci/${project.artifactId}-plugin</url>
 		<tag>HEAD</tag>
 	</scm>
+	<dependencyManagement>
+		<dependencies>
+			<dependency>
+				<groupId>io.jenkins.tools.bom</groupId>
+				<artifactId>bom-2.375.x</artifactId>
+				<version>2198.v39c76fc308ca</version>
```

Comment:
Are you sure this is the correct version to use?

Reply:
Yes, it's the [most recent release](https://repo.jenkins-ci.org/artifactory/public/io/jenkins/tools/bom/bom-2.375.x/) of `bom-2.375.x`, which corresponds to the `jenkins.version` above. I see you have dependabot enabled, so that will take care of bumping this version on new releases.

The recommended baseline changes over time and older artifacts stop being published. Ideally sometime in the next quarter we'd bump `jenkins.version` to `2.387.3` and this bom to the latest release of `bom-2.387.x`.

Happy to send a PR for that down the road if it helps.